src/Redux/books/books.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
- [ ] the **redux/book** should be empty and this component should be added in the components folder + prop.


![kkk](https://user-images.githubusercontent.com/61562626/142185363-7c34e95c-46f3-4b62-beb6-7117d1c66db3.jpeg)

Reply:
Hello dasileker,
Thank you for your feedback, and I understand changes required but according to the project requirement some requirements will be applied in the next project, here was just initialization